core/services/ocr2/delegate.go

```diff
@@ -827,7 +806,140 @@ func (d *Delegate) newServicesOCR2Keepers(
 	lc ocrtypes.LocalConfig,
 	ocrLogger commontypes.Logger,
 ) ([]job.ServiceCtx, error) {
-	keeperProvider, rgstry, encoder, logProvider, err2 := ocr2keeper.EVMDependencies20(jb, d.db, lggr, d.chainSet, d.pipelineRunner)
+	switch jb.KeeperSpec.ContractVersion {
+	case "v2.1":
```

Comment:
Is this string correct?

Reply:
Upto us how to define. I'm fine with this for now

Reply:
Can we also add a case for v2.0 and still have a default to v2.0